lib/ProductOpener/Display.pm

```diff
@@ -1345,6 +1346,10 @@ sub display_list_of_tags($$) {
 				url => format_subdomain($subdomain) . $product_link,
 				products => $products + 0, # + 0 to make the value numeric
 			};
+			
+			if (defined $linkeddata) {
+				$tagentry->{linkeddata} = $linkeddata;
```

Comment:
I'm not sure if `linkeddata` is the best name for this property, though. The ProductOpener source calls the hash `$properties` , but a `properties` property could be too generic. Any ideas?

Reply:
same_as_wikidata_qid
same_as_url  ?
what do you think @stephanegigandet

Reply:
Maybe just `sameAs`, as it's used in [schema.org](http://schema.org/sameAs)?